lib/resources/customers.ts

```diff
@@ -1,10 +1,11 @@
 'use strict'
 
-const { normalizeNotes } = require('../utils/razorpay-utils')
+import { normalizeNotes } from '../utils/razorpay-utils';
+import { ObjectWithNotes } from '../types';
 
-module.exports = function (api) {
+export default function (api) {
   return {
-    create(params, callback) {
+    create(params: ObjectWithNotes, callback) {
```

Comment:
Type for `callback`?

Reply:
Will provide once it starts throwing errors, which is in the next release. Getting rid of callbacks anyway.